### PR TITLE
add group review icon, it becomes red when group review required

### PR DESCRIPTION
**Pull request by ArtjomsPorss**
# Description
1. Added icon in Groups view which opens Group Review page.
2. The icon becomes red and text changes to "Group Review is required" when last review was more than 80% of min Review/Expiry days ago (set in settings).
3. Moved logic that determines whether a review is required from RoleRow to ReviewUtils, as it is now also used by GroupRow

# Contribution Checklist:
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have read the [contribution guidelines](https://github.com/AthenZ/athenz/blob/master/CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request.**

## Attach Screenshots (Optional) 
image

### CODE REVIEW

File: ui/src/components/role-and-group-review/RoleAndGroupReview.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { PERCENTAGE_OF_DAYS_TILL_NEXT_REVIEW } from '../constants/constants';
```
Comment (chandrasekhar1996) [inline ui/src/components/role-and-group-review/RoleAndGroupReview.js:1]: can we move this to utils directory

Comment (ArtjomsPorss) [reply ui/src/components/role-and-group-review/RoleAndGroupReview.js]: done

**Review by chandrasekhar1996 — APPROVED** [review]

File: ui/src/__tests__/components/utils/ReviewUtils.test.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import moment from 'moment-timezone';
```
Comment (abvaidya) [inline ui/src/__tests__/components/utils/ReviewUtils.test.js:1]: please add license text on all new files

File: ui/src/components/utils/ReviewUtils.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { PERCENTAGE_OF_DAYS_TILL_NEXT_REVIEW } from '../constants/constants';
```
Comment (abvaidya) [inline ui/src/components/utils/ReviewUtils.js:1]: please add license text on all new files

**Review by abvaidya — CHANGES_REQUESTED** [review]

File: ui/src/__tests__/components/utils/ReviewUtils.test.js (hunk 2 above)
Comment (ArtjomsPorss) [reply ui/src/__tests__/components/utils/ReviewUtils.test.js:16]: done

File: ui/src/components/utils/ReviewUtils.js (hunk 3 above)
Comment (ArtjomsPorss) [reply ui/src/components/utils/ReviewUtils.js:16]: done

**Review by abvaidya — APPROVED** [review]

**Pull request merged**